common/src/main/java/earth/terrarium/cadmus/common/commands/claims/ClaimSettingsCommand.java

```diff
@@ -245,6 +259,15 @@ private static ArgumentBuilder<CommandSourceStack, LiteralArgumentBuilder<Comman
             });
     }
 
+    private static TriState getInputState(String input) throws CommandRuntimeException {
+        return switch (input.toLowerCase()) {
```

Comment:
When using `toLowerCase` use `Locacle.ROOT` or it will break in languages like Turkish that does character replacements